### PR TITLE
feature: port mapping for cri manager

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: YaoZengzeng <yaozengzeng@zju.edu.cn>

### Ⅰ. Describe what this PR did

This PR implement port mapping in CRI manager by using CNI port map plugin.

### Ⅱ. Does this pull request fix one issue?

fixes part of #635

### CODE REVIEW

**Comment by YaoZengzeng** [comment]
@allencloud PTAL.

File: daemon/mgr/cri_network.go [hunk 1]
```diff
@@ -26,7 +27,7 @@ type CniMgr interface {
 	TearDownPodNetwork(config *runtime.PodSandboxConfig) error
 
 	// GetPodNetworkStatus is the method called to obtain the ipv4 or ipv6 addresses of the pod sandbox.
-	GetPodNetworkStatus(config *runtime.PodSandboxConfig) (string, error)
+	GetPodNetworkStatus(netnsPath string) (string, error)
```
Comment (allencloud) [inline daemon/mgr/cri_network.go]: I found that in previous code, we have `ip, err = c.CniMgr.GetPodNetworkStatus(netnsPath)`, then it seems that the first returned value has a type of `IP`.
So, `GetPodNetworkStatus ` gets a type of `IP` and it is a little bit weird. This is one part of interface, maybe we need to design it more carefully. @rudyfly Could you help to take a look at this?

Comment (YaoZengzeng) [reply daemon/mgr/cri_network.go]: @allencloud It's weird for me as well. 

Actually I try to be consistent with underlying interfaces 
https://github.com/alibaba/pouch/pull/730/files#diff-72a60df7c3a977ca85df89c13d782defR114

**Comment by allencloud** [comment]
LGTM

**Pull request merged**